File: Basic/JavaScript/js10.pl
use strict;
use warnings;
use integer;

my @x = (65..122);
foreach my $a1 ( @x ) {
    foreach my $a2 ( @x ) {
        foreach my $a3 ( @x ) {
            foreach my $a4 ( @x ) {
                foreach my $a5 ( @x ) {
                    print map(chr, @{[ $a1, $a2, $a3, $a4, $a5 ]}), "\n"
                        if $a1 * 3 + $a2 * 3 + $a3 * 4 + $a4 * 5 + $a5 * 6 == 2308;
                }
            }
        }
    }
}

File: Basic/JavaScript/js10.py
x = range(65,123)
for a1 in x:
    for a2 in x:
        for a3 in x:
            for a4 in x:
                for a5 in x:
                    if a1 * 3 + a2 * 4 + a3 * 5 + a4 * 6 + a5 * 7 == 2308:
                        print ''.join(map(chr, [a1, a2, a3, a4, a5]))
